### PR TITLE
#1 - Different version depending on github event

### LINKED ISSUE #1: bump minor when merging PR

### PR DESCRIPTION
**Pull request by kcarlson**
closes #1

### CODE REVIEW

File: .github/workflows/bytemotion-node-ci.yml [hunk 1]
```diff
@@ -43,16 +43,32 @@ jobs:
...
+          git config --global user.name "${{ secrets.GIT_USER_NAME }}"
           npm install
           npm run build
           git checkout -- .
           npm version $RELEASE
-          echo "Pushing to $BRANCH"
+          echo "Pushing $RELEASE to $BRANCH"
           git push origin $BRANCH
           git push origin $BRANCH --tags
           npm publish
```
Comment (wooyey) [inline .github/workflows/bytemotion-node-ci.yml:80]: What I'm lacking is to have option with changing "major" version. But I don't get whole concept behind this and... I suppose this should be done manually in `package.json` right? 
BTW some time ago I seen option which uses keywords in commit message to manual steering of these version numbers things...

**Review by wooyey — APPROVED** [review]
LGTM

File: .github/workflows/bytemotion-node-ci.yml (hunk 1 above)
Comment (kcarlson) [reply .github/workflows/bytemotion-node-ci.yml:80]: Yeah, i was thinking about that too. Major seems like something that should be done manually? Yeah, maybe the key word is a good option. Major versions are rare though and some of these libraries might never actually have another major release :smile:

Comment (kcarlson) [reply .github/workflows/bytemotion-node-ci.yml:80]: So I think the best way to determine a major release is when creating one manually through githubs formal release mechanism. I've added it, what do you thin @wooyey

Comment (wooyey) [reply .github/workflows/bytemotion-node-ci.yml:80]: Seems legit to me. Now only some place to document this briefly, at least for newcomers... and myself for some time ;)

**Review by wooyey — APPROVED** [review]

File: .github/workflows/bytemotion-node-ci.yml (hunk 1 above)
Comment (kcarlson) [reply .github/workflows/bytemotion-node-ci.yml:80]: nice! I added a readme with descriptions of the conditions necessary for each release type

**Review by tjonsson — APPROVED** [review]

**Pull request merged**